docs/getting_started/basic_usage/basic_usage.rst

```diff
@@ -134,69 +125,16 @@ log_verbosity
     An integer. Common values are 3 or 4. Starting with 5, you will get an output per mini-batch.
 
 
-There are much more params, and more details to many of the listed ones.
-See the code for more details.
-All config params can also be passed as command line params.
-See the code for some usage. The generic form is ``++param value``.
+There are much more params, and more details to many of the listed ones. Details on the parameters can be found in the
+:ref:`parameter reference <parameter_reference>`. As the reference is still incomplete, please watch out for additional
```

Comment:
There should be a newline after the dot (`.`), such that diffs/changes are nicer and more structured.

Reply:
I now went through all files and changed it to at most one sentence per line